### DISCUSSION TITLE
Local timezone

### DISCUSSION THREAD

**Discussion by bearded-papa** [garethgeorge/backrest#197] [Q&A] [answered]
Is there a way to set my local timezone as I have just done a manual backup and although I have set a TZ Variable in the compose file it said that I had started the backup at 0415 when my local time in the UK is 0715.

**Comment by garethgeorge** [comment, marked as answer]
Hey, which image are you using? Is this garethgeorge/backrest:latest?

Timezone is known to be an issue with the garethgeorge:scratch issue but should work correctly with garethgeorge/backrest:latest

Make sure you set the TZ env var e.g. `TZ=America/Los_Angeles`

**Reply by bearded-papa** [reply]
I figured out that the time on my server wasn't syncing so was out by a few hours
